### ISSUE TITLE
Table headers not readable in dark mode

### ISSUE THREAD

**Issue by nbarry10** [unl/wdntemplates#1456]
![table-headers-dark-mode](https://user-images.githubusercontent.com/5738813/91898882-069b1b80-ec62-11ea-826f-3c5300ca8e40.png)

**Comment by nbarry10** [comment]
We identified this one was coming from a DataTables css style.  This one can be closed.

**Comment by skoolbus39** [comment]
Thanks for the follow-up, @nbarry10.
